tests/functional/external_mode/test_virtualization_storageclass.py

```
@@ -0,0 +1,127 @@
+        # 3. Force Software Emulation (Safety for vSphere/Cloud labs)
+        cnv_installer.enable_software_emulation()
+
+        yield  # Execute the test
+
+        # --- TEARDOWN ---
+        log.info("Test finished. Initiating CNV Uninstallation (Teardown)...")
+        try:
+            # Cleans up HyperConverged, Subscription, CSV, and Namespace
+            cnv_installer.uninstall_cnv(check_cnv_installed=True)
+            log.info("CNV uninstallation completed successfully.")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)